upgrade-tidb-using-tiup.md

```diff
@@ -7,23 +7,24 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
 
 本文档适用于以下升级路径：
 
-- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.2 及后续修订版本。
-- 使用 TiUP 从 TiDB 5.0 版本升级至 TiDB 5.2 及后续修订版本。
-- 使用 TiUP 从 TiDB 5.1 版本升级至 TiDB 5.2 及后续修订版本。
+- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.3 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.0 版本升级至 TiDB 5.3 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.1 版本升级至 TiDB 5.3 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.2 版本升级至 TiDB 5.3 及后续修订版本。
```

Comment:
@seiya-annie PTAL

Reply:
yes, according to test, tiup supported to upgrade from 4.0,5.0,5.1,5.2 to 5.3